### PR TITLE
Add a Jupyter Notebook as an example

### PR DESCRIPTION
**Pull request by jeanboutros**

### CODE REVIEW

File: The_Python_Communist_Revolution_Notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+        "    uncompressed_string = str(base64.decodebytes(zlib.decompress(compressed_data)), encoding='ascii')\n",
+        "    return uncompressed_string\n",
```
Comment (HacKanCuBa) [inline The_Python_Communist_Revolution_Notebook.ipynb:119]: WTH is this!?
It looks like imperialist spy, or sabotage!

Comment (jeanboutros) [reply The_Python_Communist_Revolution_Notebook.ipynb:119]: It's just some song lyrics, print it out and you'll see.
Don't worry, there isn't any know python exploits that use simple strings.

Comment (jokteur) [reply The_Python_Communist_Revolution_Notebook.ipynb:119]: I have checked, they are effectively lyrics.

Comment (jeanboutros) [reply The_Python_Communist_Revolution_Notebook.ipynb:119]: I'm overriding the __str__ function of the class to be able to perform print(OrdinaryPeople()) and each class returns something different.
The class WorkingClassHeroes() when printed, returns John Lennon's song Working Class Hero but I compressed the string to make it like an easter egg.
You can check in the notebook the section that invites you to guess what the WorkingClassHeroes class prints and to uncomment the line to print the song.

**Comment by jokteur** [comment]
As the self proclaimed maintainer, I declare that your contributions are valuable for the continuation of the communist society. 

However, I cannot merge the PR until the typos have been corrected. Please correct before I can say GULAG.

**Comment by jeanboutros** [comment]
I am honoured, comrade. 
The typos have been crushed.

**Comment by bree29** [comment]
Anarchy is not communism. Are you trying to bring chaos in the revolution ?

It is very often considered good practice to squash commits for PR.

**Comment by jeanboutros** [comment]
> 

You speak like a capitalist in disguise who is conspiring against the party.

**Pull request merged**